src/commands/registries/logInToDockerCli.ts

```diff
@@ -30,7 +30,7 @@ export async function logInToDockerCli(context: IActionContext, node?: RegistryT
     }
 
     if (!username || !password) {
-        ext.outputChannel.appendLine(localize('vscode-docker.commands.registries.logIn.skipping', 'WARNING: Skipping login for "{0}" because it does not require authentication.', creds.registryPath));
+        ext.outputChannel.warn(localize('vscode-docker.commands.registries.logIn.skipping', 'WARNING: Skipping login for "{0}" because it does not require authentication.', creds.registryPath));
```

Comment:
Nit: since VSCode prepends the log level to each line we can just take the "WARNING" out of the string
```suggestion
        ext.outputChannel.warn(localize('vscode-docker.commands.registries.logIn.skipping', 'Skipping login for "{0}" because it does not require authentication.', creds.registryPath));
```

Reply:
Removed the redundant label